sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py

```diff
@@ -158,11 +158,15 @@ def _from_cncf_events(event):
         raise_with_traceback(ValueError, msg, err)
 
 
-def _build_request(endpoint, content_type, events):
+def _build_request(endpoint, content_type, events, **kwargs):
```

Comment:
Nitpick: why not an optional keywords-only argument

Reply:
changed it